message/messaging_service.hh

```diff
@@ -206,11 +206,10 @@ enum class messaging_verb : int32_t {
     TABLE_LOAD_STATS = 77,
     ESTIMATE_SSTABLE_VOLUME = 78,
     SAMPLE_SSTABLES = 79,
-    TABLET_REPAIR_COLOCATED = 80,
-    REPAIR_UPDATE_COMPACTION_CTRL = 81,
-    REPAIR_UPDATE_REPAIRED_AT_FOR_MERGE = 82,
-    WORK_ON_VIEW_BUILDING_TASKS = 83,
-    LAST = 84,
+    REPAIR_UPDATE_COMPACTION_CTRL = 80,
+    REPAIR_UPDATE_REPAIRED_AT_FOR_MERGE = 81,
+    WORK_ON_VIEW_BUILDING_TASKS = 82,
+    LAST = 83,
```

Comment:
The incremental repair related verbs (`REPAIR_UPDATE_COMPACTION_CTRL`, `REPAIR_UPDATE_REPAIRED_AT_FOR_MERGE `) were introduced in https://github.com/scylladb/scylladb/commit/0d7e518a26127189eb03b7635df7f37ab93fc297, which will be released for the first time in 2025.4.

`WORK_ON_VIEW_BUILDING_TASKS` is related to the view build coordinator, which will also go live in 2025.4 for the first time.

So, changing these RPC's numbers is OK.